server/seeds/index.js

```diff
@@ -13,3 +14,27 @@ db.once("open", async () => {
     process.exit(1);
   }
 });
+
+db.once("open", async () => {
+  try {
+    await db.dropDatabase();
+    await User.create(userSeeds);
```

Comment:
In the same _try_ block, we should do User.create, Song.create, and Rating.create. As it is now, it will seed the Song database, then end the program without running the User or Rating part at all.